draft-02.xml

```diff
@@ -969,10 +969,25 @@
             'safe', 'unsafe', or 'idempotent.'
           </t>
           <t>
-            The 'rt' attribute is OPTIONAL and is an opaque string and MAY match the
-            <xref target="prop-id" format="title"/>
-            of an
-            existing<xref target="prop-descriptor" format="title"/>.
+            The 'rt' attribute is OPTIONAL and SHOULD be one of the following:
+            <list style="numbers">
+              <t>
+                A fragment identifier (e.g. rt="#user") which points
+                to the <xref target="prop-id" format="title"/>
+                of an existing <xref target="prop-descriptor" format="title"/>
+                in the current ALPS document.
+              </t>
+              <t>
+                A resolvable URL (e.g. rt="http://example.org/profiles/user")
```

Comment:
@fosrias 

we can say "it SHOULD|MUST match..." (instead of the current MAY) but do you want to say it MUST|SHOULD|MAY contain a fragment identifier?

IOW, is dropping the second option all that is needed here? or is there more that needs to be added/changed?